VAMobile/src/components/AnnouncementBanner.tsx

```diff
@@ -61,9 +61,8 @@ const AnnouncementBanner: FC<AnnouncementBannerProps> = ({ title, link, a11yLabe
             width={24}
             height={24}
```

Comment:
Just noting this is redundant as the Icon component defaults to 24x24 if no height/width is sent.

Reply:
Thanks for pointing that out, just removed this